Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.py

```diff
@@ -954,7 +966,7 @@ def main():
                                       incorrect_fields=incorrect_fields)
 
     # Dumps all dict in the current incident
-    incident_df = dumps_json_field_in_incident(incident)
+    incident_df = dumps_json_field_in_incident(deepcopy(incident))
```

Comment:
this fixes an issue with nested fields handling (we do it only for the current incident)